docs/src/makielayout/grids.md

```diff
@@ -286,6 +286,43 @@ save("example_trimming_3.svg", scene); nothing # hide
 ![trimming 3](example_trimming_3.svg)
 
 
+## Tweaking space between rows and columns
+
+You can use `rowgap!` and `colgap!` to change the spacing between rows or
+columns respectively.
+
+```@example spacing
+using CairoMakie
+
+scene, layout = layoutscene(resolution = (1200, 900))
+
+axs = layout[] = [Axis(scene) for i in 1:3, j in 1:3]
```

Comment:
nice, thank you! but could you switch the example to the `Figure` workflow? I assume people will mostly try to change figures and then they can see that you have to access `fig.layout` to call layout specific functions on it.

the axs part could then also be the new syntax `axs = [Axis(f[i, j]) for i in 1:3, j in 1:3]`

Reply:
Done.